### PR TITLE
Fast in-loop MCQA

### PR DESCRIPTION
**Pull request by davidheineman**
Bump in-loop evals to v0.8.1. This will add "fast" MCQA, which performs MC tasks in 1 forward pass instead of 4 forward passes. (We extract the A/B/C/D logits from a single pass).

https://github.com/allenai/OLMo-in-loop-evals/pull/8

This will make the MC tasks 4x faster, and produces the same numbers.

Also, added Java, Rust and C++ translated MBPP BPB.

### CODE REVIEW

**Review by epwalsh — APPROVED** [review]
Sweet!
